RFCs/0097-declarativeCDK

```diff
@@ -0,0 +1,48 @@
+# Declarative CDK
+
+* **Original Author(s):**: @evgenyka
+* **Tracking Issue**: #97
+
+# Summary
+
+Introducing Declarative CDK - a project that allows non-developer customers use CDK Constructs in their CloudFormation templates. 
+
+# Motivation
+
+The customers will benefit from an abstraction level that AWS CDK Constructs provide and ability to reuse existing Constructs content. It will result in the modularity of the CloudFormation templates, which will minimize the risks of ending up with template errors and also makes it easier to spot errors, if any.
```

Comment:
> It will result in the modularity of the CloudFormation templates

If the goal is modularity, shouldn't we be looking at improvements to how modules work instead?

> which will minimize the risks of ending up with template errors

How?

> and also makes it easier to spot errors, if any

How?

Reply:
The advantage of construct library is that there is a lot of canned content that already exists. With Modules there is no content that is ready to use. So, using CDK Constucts library give us a head start on any other solution